test/verify/check-machines-dbus

```diff
@@ -2097,9 +2098,13 @@ class TestMachinesDBus(machineslib.TestMachines):
                     b.click("#vm-subVmTest1-networks") # open the Network Interfaces subtab
                 else:
                     b.click("#delete-vm-subVmTest1-iface-{0}".format(self.nic_num))
-                    # Confirm in confirmation windown
+                    # Confirm in confirmation window
                     b.wait_in_text(".modal-dialog .modal-header .modal-title", "Delete Network Interface")
+                    vm_state = b.text("#vm-subVmTest1-state")
                     b.click(".modal-footer button:contains(Delete)")
+                    # On a running VM detaching NIC takes longer and we can see the spinner
+                    if vm_state == "running":
+                        b.wait_present(".modal-footer .spinner")
```

Comment:
Isn't this and invitation for race condition? (if device gets detached too fast)

Reply:
It's time specific indeed but it failed 0 times and there is no other way to test it. I suggest to keep it and if it ends up being flaky we drop it.